main.py

```diff
@@ -22,3 +38,36 @@ def include_file(filename, start_line=0, end_line=None):
             lines = f.readlines()
         line_range = lines[start_line:end_line]
         return ''.join(line_range)
+
+    @env.macro
+    def cards(pages, columns=1):
+        if isinstance(pages, str):
+            pages = [pages]
+        cards_markdown = []
+        for page in pages:
+            with open("docs/" + page, "r") as doc_file:
+                doc = doc_file.read()
+                match = re.search("^# (.*)", doc, re.MULTILINE)
+                if match:
+                    header = match.groups()[0]
+                else:
+                    header = ""
+                default_meta = {
+                    "title": header,
+                    "short": "",
+                    "description": ""
+                }
+                doc_meta = {
+                    **default_meta,
+                    **meta.get_data(doc)[1]
+                }
+                cards_markdown.append(
+                    CARDS_MARKDOWN_TEMPLATE % (
+                        doc_meta['short'] or doc_meta['title'],
+                        doc_meta['description'] or doc_meta['title'],
+                        doc_meta['title'],
+                        "/" + page
```

Comment:
The generated link is missing the `en/<version>` part. `<version>` is resolved by readthedocs to be the current version of the doc.